### ISSUE TITLE
ITIS - Incorrect synonyms - Chiridopsis bipunctata (Linnaeus, 1767)

### ISSUE THREAD

**Issue by ManonGros** [CatalogueOfLife/data#824]
**Describe the problem**:
This is based on a feedback message received on the GBIF Helpdesk.

> Cryptocephalus bipunctatus is a transpalearctic species from the Cryptocephalinae subfamily, widespread in Europe, while Chiridopsis bipunctata is an Indian species from the Cassidinae subfamily. Not to mention that they don't even look anything alike:

Cryptocephalus bipunctatus (Linnaeus, 1758): https://www.gbif.org/occurrence/4852794051

Chiridopsis bipunctata (Linnaeus, 1767): https://www.gbif.org/occurrence/4908934166

**Link to effected CoL webpages**:
https://www.catalogueoflife.org/data/taxon/69SZM

**Comment by yroskov** [comment]
@DaveNicolson, I think, it's might be for you. (If it's not the ITIS species, my apology. CoL & CLB are misbehaving today, and I am not able to confirm data provider for the species).

**Comment by yroskov** [comment]
Yes, you are right. CoL also does not have _Cryptocephalus bipunctatus_.

**Comment by mdoering** [comment]
@ManonGros this is a homotypic grouping problem from the GBIF Backbone. @camiplata @DianRHR please check what the XR does on those and make use of the `basionymExclusions` setting in the xCfg if you need to.

**Comment by DianRHR** [comment]
both names are fine and separate as accepted species. 
Cryptocephalus bipunctatus is merging under Cryptocephalini (from UKSI)
Chiridopsis bipunctata under Cassidini ( provided by ITIS in the baseCOL)
